lib/profile/profile_screen.dart

```diff
@@ -156,7 +156,7 @@ class _ProfileScreenState extends State<ProfileScreen> {
                                 ? 8
                                 : 0),
                         child: Padding(
-                          padding: EdgeInsets.all(16),
+                          padding: EdgeInsets.all(8),
```

Comment:
Why 8? Other CardView have 16 spacing? Could we manage to keep 16?

Reply:
The thing is, in the card just above ("add comment"), you "cheated" by using a Stack so the 16 margin does not apply to the "Save" icon.
So, if I left this one at 16, the "PopupMenu" icon and the Save icon would be grossly misaligned

Reply:
Then could we add margin to text so text from above cell is align?